DOC-6147 description/difficulty for list and time series code examples (#2628)
redis/docs: content/develop/data-types/timeseries/_index.md

Use [`TS.GET`]({{< relref "commands/ts.get/" >}}) to retrieve the data point
with the highest timestamp in a time series. This returns both the timestamp and the value.

> {{< clients-example set="time_series_tutorial" step="get" description="Foundational: Retrieve the most recent data point from a time series using TS.GET to get the latest value and timestamp" difficulty="beginner" >}}

Reviewer: Auggie is VERY wordy today. :)

```suggestion
{{< clients-example set="time_series_tutorial" step="get" description="Foundational: Use TS.GET to get the latest value and timestamp" difficulty="beginner" >}}
```
